api-tests/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "api-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "api-tests": "vitest --config vitest.config.ts $VERSION",
+    "lint": "eslint tests/**/*.ts",
+    "compile": "tsc -p ./tsconfig.json --noEmit --skipLibCheck",
```

Comment:
```suggestion
    "check-types": "tsc",
```

Reply:
The noEmmit is still required, to not generate js files.